tests/Struct/FindologicProductTest.php

```diff
@@ -369,9 +368,13 @@ public function testProduct(): void
         );
 
         $salesChannel = $this->salesChannelContext->getSalesChannel();
-        $domain = $salesChannel->getDomains()->first()->getUrl();
+        $domains = $salesChannel->getDomains();
```

Comment:
Is there already a `default.headless` included?

Reply:
Yes that's the Headless Sales Channel, which is always created by default. Only recently with Shopware 6.4 it appears to also to include an URL which is not really an URL, so we just filter all of these headless Sales Channels out of this.